pkg/api/v1/resource/helpers.go

```diff
@@ -193,7 +194,28 @@ func ExtractContainerResourceValue(fs *v1.ResourceFieldSelector, container *v1.C
 	case "requests.ephemeral-storage":
 		return convertResourceEphemeralStorageToString(container.Resources.Requests.StorageEphemeral(), divisor)
 	}
-
+	// handle extended standard resources with dynamic names
```

Comment:
Slightly tangential question from me - it appears there are ~3 separate `ExtractContainerResourceValue` functions that we are updating here. Is there the opportunity to consolidate those into a single function which is reused? It feels less than optimal that we have to add the same 20 lines of code to three different functions.

Very possible I'm missing an important difference between them though :)